src/Zomp.SyncMethodGenerator/UserMappings.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Zomp.SyncMethodGenerator;
+
+internal sealed class UserMappings(

```

Comment:
Maybe I should rename this class to `UserMappingResult` or something else.
Plural class names are kinda cursed 🙂